handlers_test.go

```diff
@@ -176,8 +176,28 @@ func TestHashHandler(t *testing.T) {
 	}
 }
 
+func TestReadiness(t *testing.T) {
+	cfg := defaultCfg
+	cfg.WaitForApp = false
```

Comment:
And `TestReadyHandler` already covers the `true` case, although maybe it should also get a similar explicit setting for this switch?

Reply:
Fixed in ed2fea3.